src/framework/audio/imixer.h

```diff
@@ -58,14 +55,17 @@ class IMixer : virtual public IAudioSource
     virtual unsigned int addChannel(std::shared_ptr<IAudioSource> source) = 0;
     virtual void removeChannel(unsigned int channelId) = 0;
 
+    //! mixed source
+    virtual std::shared_ptr<IAudioSource> mixedSource() = 0;
```

Comment:
Please, add const to this method

Reply:
unfortunately impossible, does not compile